### PR TITLE
[13.0] Add packaging for sale orders and wishlists

### PR DESCRIPTION
**Pull request by simahawk**
TODO

- [x] add dependency on https://github.com/OCA/sale-workflow/pull/1114 and https://github.com/OCA/stock-logistics-warehouse/pull/918
- [x] depends on https://github.com/OCA/stock-logistics-warehouse/pull/931
- [x] depends on https://github.com/OCA/sale-workflow/pull/1192
- [x] finish tests

### CODE REVIEW

**Comment by simahawk** [comment]
~~TODO: fix base tests~~

File: shopinvader_sale_packaging/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        Sell products by packaging.
+    """,
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Camptocamp SA",
+    "website": "https://github.com/shopinvader/odoo-shopinvader",
+    "depends": [
+        "shopinvader",
+        "sale_stock",
+        "sale_order_line_packaging_qty",
```
Comment (grindtildeath) [inline shopinvader_sale_packaging/__manifest__.py]:
```suggestion
        "sale_by_packaging",
```

**Review by grindtildeath — COMMENTED** [review]
Small change related to https://github.com/OCA/sale-workflow/pull/1191

**Comment by guewen** [comment]
The last dependency is merged :)

**Comment by simahawk** [comment]
@guewen removed, mind to drop a review? :)

**Review by guewen — APPROVED** [review]

**Review by i-vyshnevska — APPROVED** [review]

**Pull request merged**